testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/model/UserSessionProviderOfflineTest.java

```diff
@@ -462,10 +463,8 @@ public void testExpired(KeycloakSession session) {
 
                 // Persist 3 created userSessions and clientSessions as offline
                 testApp[0] = realm.getClientByClientId("test-app");
-                List<UserSessionModel> userSessions = currentSession.sessions().getUserSessions(realm, testApp[0]);
-                for (UserSessionModel userSession : userSessions) {
-                    offlineSessions.put(userSession.getId(), createOfflineSessionIncludeClientSessions(currentSession, userSession));
-                }
+                currentSession.sessions().getUserSessionsStream(realm, testApp[0]).collect(Collectors.toList())
```

Comment:
Again collectToMap + putAll

Reply:
Same as above. Test fails because it cannot create the offline sessions while the stream is still "live"